### DISCUSSION TITLE
Constraints in map display are different than in the MCDU

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3427] [Help]
I noticed today that the constraints for waypoints on a STAR were different in the MCDU vs the front Plan map. Anyone know if this is my fault and can be fixed? Thx

![constraints](https://user-images.githubusercontent.com/75288729/107280268-1ba48380-6a26-11eb-8944-cc9ba7fc21f4.jpg)

**Comment by robinottawa** [comment]
No answers. I stumped everyone.

**Reply by tracernz** [reply]
New flightplan manager is in progress so unlikely anybody will spend their time looking deeply at the old Asobo one. If you want to though nobody will stop you from submitting a patch.
